x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/alert_hooks.ts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
We should move this file to the `view/alerts` folder, as hooks are associated with views. Also I think it might be better to rename the file to something like `use_alerts_selector.ts` because we will probably add more hooks in the future and I don't think we should keep them all in one file.